Gruntfile.js

```diff
@@ -33,6 +33,7 @@ module.exports = function(grunt) {
       default_options: {
         options: {
           path: 'test/js/*.html',
+          cwd: '../..',
```

Comment:
This is not the default option present in `grover.js`  
https://github.com/leo-guinan/grunt-grover/blob/master/tasks/grover.js#L81